pps_mw_training/pipelines/cloud_base/training_data.py

```diff
@@ -0,0 +1,142 @@
+from pathlib import Path
+from typing import Any
+import json
+import numpy as np  # type: ignore
+import tensorflow as tf  # type: ignore
+import xarray as xr  # type: ignore
+from pps_mw_training.utils.scaler import get_scaler
+
+
+def _load_data(
+    data_files: np.ndarray,
+    input_parameters: str,
+    label_parameters: str,
+    fill_value_input: float,
+    fill_value_label: float,
+) -> list[np.ndarray]:
+    """Load, scale, and filter data."""
+    with xr.open_mfdataset(
+        [f.decode("utf-8") for f in data_files],
+        combine="nested",
+        concat_dim="nscene",
+    ) as all_data:
+        input_params = json.loads(input_parameters)
+        label_params = json.loads(label_parameters)
+        input_scaler = get_scaler(input_params)
+        input_data = np.stack(
+            [
+                input_scaler.apply(all_data[p["name"]].values, idx)
+                for idx, p in enumerate(input_params)
+            ],
+            axis=3,
+        )
+        label_scaler = get_scaler(label_params)
+        label_data = np.stack(
+            [
+                label_scaler.apply(all_data[p["name"]].values, idx)
+                for idx, p in enumerate(label_params)
+            ],
+            axis=3,
+        )
+        input_data[~np.isfinite(input_data)] = fill_value_input
+        label_data[~np.isfinite(label_data)] = fill_value_label
+        return [
+            np.float32(input_data),
+            np.float32(label_data),
+        ]
+
+
+@tf.function(
+    input_signature=(
+        tf.TensorSpec(shape=(None,), dtype=tf.string),
+        tf.TensorSpec(shape=(), dtype=tf.string),
+        tf.TensorSpec(shape=(), dtype=tf.string),
+        tf.TensorSpec(shape=(), dtype=tf.float32),
+        tf.TensorSpec(shape=(), dtype=tf.float32),
+    )
+)
+def load_data(
+    matched_files,
```

Comment:
replace matched_files to files, "matched" is a bit confusing name in this module, I guess it is a heritage from the other pipeline where it make more sense. This applies to more functions in this module.

Reply:
Fixed